src/main/php/cfg/verb/verb.php

```diff
@@ -972,15 +979,15 @@ private function add(sql_db $db_con): string
                     $db_rec->name = $this->name;
                     $db_rec->usr = $this->usr;
                     // save the verb fields
-                    $result .= $this->save_fields($db_con, $db_rec);
+                    foreach ($this->save_fields($db_con, $db_rec) as $msg) {
+                        $usr_msg->add($msg);
+                    }
                 }
 
             } else {
-                $result .= "Adding verb " . $this->name . " failed.";
+                $usr_msg->add_id_with_vars(msg_id::VERB_ADD_FAILED, [msg_id::VAR_NAME => $this->name]);
             }
         }
-
-        return $result;
```

Comment:
I guess, the $usr_msg object should be returned here, so that the calling function can pick it up and show it to the user.